cw_wownero/ios/Classes/wownero_api.h

```diff
@@ -0,0 +1,36 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include "CwWalletListener.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+bool create_wallet(char *path, char *password, char *language, int32_t networkType, char *error);
+bool restore_wallet_from_seed(char *path, char *password, char *seed, int32_t networkType, uint64_t restoreHeight, char *error);
```

Comment:
Hey, why does this header still use the restoreHeight parameter if the `restore_wallet_from_seed` function in `wownero_api.cpp` no longer does?

Reply:
Also, why doesn't it matter?  This code all works, but when adding `validate_address` I noticed these headers were off.